bign256/README.md

```diff
@@ -0,0 +1,88 @@
+# [RustCrypto]: BIGN P-256 (bign-curve256v1) elliptic curve
+
+[![crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+[![Build Status][build-image]][build-link]
+![Apache2/MIT licensed][license-image]
+![Rust Version][rustc-image]
+[![Project Chat][chat-image]][chat-link]
+
+Pure Rust implementation of the BIGN P-256 (a.k.a. bign-curve256v1) elliptic curve
+with support for ECDSA signing/verification, and general purpose curve
+arithmetic support implemented in terms of traits from the [`elliptic-curve`]
+crate.
+
+[Documentation][docs-link]
+
+## ⚠️ Security Warning
+
+The elliptic curve arithmetic contained in this crate has never been
+independently audited!
+
+This crate has been designed with the goal of ensuring that secret-dependent
+operations are performed in constant time (using the `subtle` crate and
+constant-time formulas). However, it has not been thoroughly assessed to ensure
+that generated assembly is constant time on common CPU architectures.
+
+USE AT YOUR OWN RISK!
+
+## Supported Algorithms
+
+- [Elliptic Curve Digital Signature Algorithm (ECDSA)][ECDSA]: gated under the
+  `ecdsa` feature.
```

Comment:
It doesn't look like you've wired up an ECDSA implementation yet (e.g. by adding an `ecdsa` module which defines curve-specific type aliases)

Reply:
It is necessary to add ECDSA, or I can add it later?

Reply:
You can add it later, but in the meantime it would probably be good to remove that from the documentation until it's implemented